### DISCUSSION TITLE
How can I change the path environment variable in the powershell startup options?

### DISCUSSION THREAD

**Discussion by YexuanXiao** [PowerShell/PowerShell#21014] [Q&A] [answered]
I use windows terminal and powershell, and I am a bit of a neat freak. I want to be able to switch my development environment by switching profiles in windows terminal. I used to do this, but unfortunately I lost this technique when I reinstalled my system.

**Comment by 237dmitry** [comment]
Add to `$profile`:

```powershell
$env:PATH += ";C:\dir1;D:\dir2"
```

**Reply by SamErde** [reply, marked as answer]
Reviving this old thread to note one caveat to this approach: it can sometimes leave you with a double semicolon if the PATH variable already ends with a semicolon. That can be avoided by using this:

```powershell
$env:PATH = ($env:PATH).TrimEnd(';') + ";C:\Dir1;C:\Dir2;C:\Dir3"
```
